scripts/ipmConfigEpics

```diff
@@ -131,11 +131,11 @@ elif [ $hutch == 'cxi' ]; then
 	BASE='CXI:DG3:BMMON'
 	EVR='CXI:DG3:BMMON:EVR'
 	WAVE8='dg3ipm'
-    elif [ $1 == 'sc1ipm' ]; then
+    elif [ $1 == 'usrdio' ]; then
         IOC="IOC:CXI:SC1:BMMON"
         BASE='CXI:SC1:BMMON'
         EVR='CXI:SC1:BMMON:EVR'
```

Comment:
@c-tsoi Can you please do a follow-up PR to edit these PVs?

Reply:
Just made the PR.